sys/cddl/contrib/opensolaris/uts/common/fs/zfs/dsl_scan.c

```diff
@@ -3413,7 +3425,8 @@ scan_exec_io(dsl_pool_t *dp, const blkptr_t *bp, int zio_flags,
 	vdev_t *rvd = spa->spa_root_vdev;
 	abd_t *data = abd_alloc_for_io(size, B_FALSE);
 	uint64_t maxinflight;
-
+	unsigned int scan_delay = 0;
+	
```

Comment:
Again extra tab.  Not to bug you, but this makes patch less readable and potentially more conflicting.

Reply:
Fixing.